ruby/cross_service_examples/item-tracker/README.md

```diff
@@ -0,0 +1,100 @@
+# Item Tracker, a Cross-Service Example using the AWS SDK for Ruby
+
+This example code comprises a "real-world" reference application showcasing a [serverless Amazon Aurora database](https://aws.amazon.com/rds/aurora/) using the [AWS SDK for Ruby](https://aws.amazon.com/sdk-for-ruby/).
+
```

Comment:
I generally prefer to link to doc pages instead of marketing pages:
* Aurora: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/CHAP_AuroraOverview.html
* Ruby SDK: https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/welcome.html